app/uk/gov/hmrc/digitalservicestax/views/govuk_wrapper.scala.html

```diff
@@ -82,7 +82,11 @@
 }
 
 @footerTop = {}
-@footerLinks = {}
+@footerLinks = {
+    @uiLayouts.footer_links(
+        accessibilityFooterUrl = Some("/digital-services-tax/accessibility-statement")
```

Comment:
I think this should be something like..

@uk.gov.hmrc.digitalservicestax.controllers.routes.JourneyController.accessibilityStatement

.. rather than hard coded